src/pcap/pcap.go

```diff
@@ -112,7 +108,7 @@ func validateDevice(name string) (err error) {
 		}
 	}()
 
-	if len(name) > 16 {
+	if len(name) > maxDeviceNameLength {
 		return fmt.Errorf("name too long: %d > 16", len(name))
```

Comment:
Don't hardcode the error string if the condition is based on a variable.
```suggestion
		return fmt.Errorf("name too long: %d > %d", len(name), maxDeviceNameLength)
```